docs/tables/aws_guardduty_finding/queries.md

```diff
@@ -0,0 +1,384 @@
+## Activity Examples
+
+### Daily Finding Trends
+
+Analyze the daily distribution of GuardDuty findings to identify security patterns and potential attack campaigns over time. This temporal analysis helps establish baseline activity and detect anomalous spikes in security events.
+
+```sql
+select
+  strftime(created_at, '%Y-%m-%d') as finding_date,
+  count(*) as finding_count
+from
+  aws_guardduty_finding
+group by
+  finding_date
+order by
+  finding_date asc;
+```
+
+```yaml
+folder: GuardDuty
+```
+
+### Top 10 Finding Types
+
+Generate a ranked list of the most prevalent GuardDuty finding types, helping security teams focus on the most common security issues affecting their infrastructure. This insight drives prioritization of security controls and remediation efforts.
+
+```sql
+select
+  type,
+  count(*) as finding_count
+from
+  aws_guardduty_finding
+group by
+  type
+order by
+  finding_count desc
+limit 10;
+```
+
+```yaml
+folder: GuardDuty
+```
+
+### Findings by Account and Region
+
+Perform a comprehensive analysis of security findings across your AWS organization, breaking down incidents by account and region while calculating average severity scores. This helps identify high-risk areas and potential security gaps in your multi-account infrastructure.
+
+```sql
+select
+  account_id,
+  region,
+  count(*) as finding_count,
+  round(avg(severity), 2) as avg_severity
+from
+  aws_guardduty_finding
+group by
+  account_id,
+  region
+order by
+  finding_count desc;
+```
+
+```yaml
+folder: GuardDuty
+```
+
+### Findings by Severity Level
+
+Strategically categorize GuardDuty findings into High (7.0-8.9), Medium (4.0-6.9), and Low (0.1-3.9) severity bands to enable risk-based prioritization and resource allocation for incident response teams.
+
+```sql
+select
+  case
+    when severity >= 7 then 'High (7.0-8.9)'
+    when severity >= 4 then 'Medium (4.0-6.9)'
+    else 'Low (0.1-3.9)'
+  end as severity_level,
+  count(*) as finding_count
+from
+  aws_guardduty_finding
+group by
+  severity_level
+order by
+  finding_count desc;
+```
+
+```yaml
+folder: GuardDuty
+```
+
+## Detection Examples
+
+### Privilege Escalation Attempts
+
+Monitor and detect potential privilege escalation activities that could indicate lateral movement attempts or compromised credentials. This query helps identify attackers trying to gain elevated access within your AWS environment.
+
+```sql
+select
+  tp_timestamp,
+  tp_index as account_id,
+  region,
+  title,
+  type,
+  severity,
+  description
```

Comment:
```suggestion
select
  tp_timestamp,
  title,
  type,
  severity,
  description,
  tp_index as account_id,
  region
```

For non-aggregated queries, can you please move the account/region info toward the end of the `select` statement? I think seeing the timestamp + finding specific info first (going left to right) helps find relevant findings easier.